speechbrain/utils/bleu.py

```diff
@@ -0,0 +1,151 @@
+import sacrebleu
+from speechbrain.dataio.dataio import merge_char, split_word
+from speechbrain.utils.metric_stats import MetricStats
+
+
+def merge_words(sequences):
```

Comment:
Could you add a small description of what this function does ? And why we need it ?